client/my-sites/site-settings/form-security.jsx

```diff
@@ -0,0 +1,80 @@
+/**
+ * External dependencies
+ */
+import React, { Component } from 'react';
+import { flowRight, partialRight, pick } from 'lodash';
+import { localize } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import wrapSettingsForm from './wrap-settings-form';
+import SectionHeader from 'components/section-header';
+import Button from 'components/button';
+import Sso from './sso';
+import QueryJetpackModules from 'components/data/query-jetpack-modules';
+
+class SiteSettingsFormSecurity extends Component {
+	renderSectionHeader( title, showButton = true, disableButton = false ) {
+		const { isRequestingSettings, isSavingSettings, translate } = this.props;
+		return (
+			<SectionHeader label={ title }>
+				{ showButton &&
+					<Button
+						compact
+						primary
+						onClick={ this.props.handleSubmitForm }
+						disabled={ isRequestingSettings || isSavingSettings || disableButton }>
+						{ isSavingSettings ? translate( 'Saving…' ) : translate( 'Save Settings' ) }
+					</Button>
+				}
+			</SectionHeader>
+		);
+	}
+
+	render() {
+		const {
+			fields,
+			handleAutosavingToggle,
+			jetpackSettingsUISupported,
+			handleSubmitForm,
+			isRequestingSettings,
+			isSavingSettings,
+			siteId,
+			translate
+		} = this.props;
+
+		if ( ! jetpackSettingsUISupported ) {
+			return null;
+		}
+
+		return (
+			<form
+				id="site-settings"
+				onSubmit={ handleSubmitForm }
+				className="site-settings__security-settings"
+			>
+				<QueryJetpackModules siteId={ siteId } />
+
+				{ this.renderSectionHeader( translate( 'WordPress.com log in' ), false ) }
```

Comment:
Hi! I've found a possible matching string that has already been translated **23** times:
`translate( 'Log in to WordPress.com' )` *ES Score: 12*
See 2 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/add/site-settings-sso-card)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).